week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,137 @@
+// When clicking ** blue ** it should change:
+
+// - Div tag with the id ** Jumbotron ** background color to`#588fbd`
+//     - ** Donate a bike ** button background color to`#ffa500`
+//         - ** Volunteer ** button background color to`black` and text color to`white`
+
+//get an element
+
+let blueBtn = document.querySelector('#blueBtn')
+
+
+
+
+let blueButtonChanged = function () {
```

Comment:
Be careful with the name of this function - think of the name of the function in terms of what action it performs. So you can say that when this function is called it will "update colour theme to blue" or something like that but better though 😛